packages/atclient/src/atclient.c

```diff
@@ -668,6 +673,7 @@ int atclient_send_heartbeat(atclient *heartbeat_conn) {
     memset(recv, 0, sizeof(unsigned char) * recvsize);
   }
   size_t recv_len = 0;
+  char *ptr = (char *)recv;
```

Comment:
this pointer isn't used

Reply:
This has been addressed in the latest commit '[80f123c](https://github.com/atsign-foundation/at_c/pull/453/commits/80f123cf7f73d1e85060a92c8240e61da5f01086)'